workflows/createpanelofnormals.nf

```diff
@@ -63,24 +78,89 @@ workflow CREATEPANELOFNORMALS {
 
     ch_versions = Channel.empty()
 
-    //
-    // SUBWORKFLOW: Read in samplesheet, validate and stage input files
-    //
-    INPUT_CHECK (
-        file(params.input)
-    )
-    ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
-    // TODO: OPTIONAL, you can use nf-validation plugin to create an input channel from the samplesheet with Channel.fromSamplesheet("input")
-    // See the documentation https://nextflow-io.github.io/nf-validation/samplesheets/fromSamplesheet/
-    // ! There is currently no tooling to help you write a sample sheet schema
+    input         = Channel.fromSamplesheet("input")
```

Comment:
I like to prefix all channels with `ch_` to make that obvious, but that might be a personal thing.